test/cql-pytest/test_describe.py

```diff
@@ -460,6 +460,31 @@ def test_generic_desc(cql, random_seed):
             assert generic_tbl == desc_tbl
             assert generic_idx == desc_idx
 
+def test_generic_desc_user_defined(scylla_only, cql, test_keyspace):
+    with new_random_type(cql, test_keyspace) as udt:
+        generic_udt = cql.execute(f"DESC {udt}")
+        desc_udt = cql.execute(f"DESC TYPE {udt}")
+
+        assert generic_udt == desc_udt
```

Comment:
stylistic nitpick (feel free to reject if you don't like it):
```python
assert cql.execute(f"DESC {udt}") = cql.execute(f"DESC TYPE {udt}")
```
is four times shorter, and avoids overloading the reader with more variable names ;-)